ext/OceananigansMakieExt.jl

```diff
@@ -41,7 +66,7 @@ grids) with NaNs;
 - transferring data from GPU to CPU if necessary.
 """
 function make_plottable_array(f)
-
+    compute!(f)
```

Comment:
so is `compute!(::Field) = nothing`?

Reply:
yes https://github.com/CliMA/Oceananigans.jl/blob/840f6026c3025872e57c66e9b134ffbeb24bdc8c/src/Fields/field.jl#L440

Reply:
great! thanks! mostly my question was a sanity check for my understanding